src/transformers/models/mamba/modeling_mamba.py

```diff
@@ -35,6 +35,7 @@
 from ...utils.import_utils import is_causal_conv1d_available, is_mamba_ssm_available
 from .configuration_mamba import MambaConfig
 
+from .pscan import pscan
```

Comment:
For such a dependency we need to protect the import! Similarly to what we do for the mamba kernels!